controllers/admin/AdminTranslationsController.php

```diff
@@ -887,6 +897,11 @@ public function submitAddLang()
                                 $this->errors[] = sprintf(Tools::displayError('Cannot delete the archive %s.'), $file);
                             }
 
+                            //fetch cldr datas for the new imported locale
+                            $languageCode = explode('-', Language::getLanguageCodeByIso($arr_import_lang[0]));
+                            $cldrUpdate = new Update(_PS_TRANSLATIONS_DIR_);
+                            $cldrUpdate->fetchLocale($languageCode[0].'-'.strtoupper($languageCode[1]));
```

Comment:
To avoid issues with multibyte chars, it could be better to use Tools::strtoupper()